Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:14.15.4-alpine3.10@sha256:fe215d05cdde4b7f2a0f546c88a8ddc4f5fa280a204acdfc2383afe901fd6d84 as build
+
+USER node
+
+WORKDIR /home/node
+
+COPY --chown=node packages/ .
+
+RUN npm --prefix frontend install frontend && \
```

Comment:
I think it is better to use `npm ci` instead of using `npm install` it makes the build reproducible, since it doesn't update the package-lock.json if it is not available.

See https://docs.npmjs.com/cli/v6/commands/npm-ci

Reply:
good idea, changed